editor/import/resource_importer_texture.cpp

```diff
@@ -218,7 +218,10 @@ void ResourceImporterTexture::get_import_options(const String &p_path, List<Impo
 	r_options->push_back(ImportOption(PropertyInfo(Variant::BOOL, "process/HDR_as_SRGB"), false));
 	r_options->push_back(ImportOption(PropertyInfo(Variant::INT, "process/size_limit", PROPERTY_HINT_RANGE, "0,4096,1"), 0));
 	r_options->push_back(ImportOption(PropertyInfo(Variant::INT, "detect_3d/compress_to", PROPERTY_HINT_ENUM, "Disabled,VRAM Compressed,Basis Universal"), (p_preset == PRESET_DETECT) ? 1 : 0));
-	r_options->push_back(ImportOption(PropertyInfo(Variant::FLOAT, "svg/scale", PROPERTY_HINT_RANGE, "0.001,100,0.001"), 1.0));
+
+	if (p_path.get_extension() == "svg") {
```

Comment:
Godot probably also supports also `svgz` files

Reply:
Apparently not anymore after #49645, bit I didn't check if it's really a lack of support or was an overzealous refactoring.

Reply:
I didn't see svgz support, so we'd probably need to decompress the svgz. Is it merely a zipped svg?

Reply:
So I checked in `3.x` with nanosvg and it doesn't seem to support svgz either. So I guess it was a bug that it's listed as a supported extension.

I'm not sure how widely supported SVGZ is, it's supposed to only be gzipped SVG but even Inkscape doesn't seem to be able to open them - the few I try after downloading them from random pages required unzipping with 7zip before Inkscape could open them.